test/ssl_test.c

```diff
@@ -233,7 +233,9 @@ static int execute_test(SSL_TEST_FIXTURE fixture)
         client_ctx = SSL_CTX_new(DTLS_client_method());
         if (test_ctx->handshake_mode == SSL_TEST_HANDSHAKE_RESUME) {
             resume_server_ctx = SSL_CTX_new(DTLS_server_method());
-            OPENSSL_assert(resume_server_ctx != NULL);
+            resume_client_ctx = SSL_CTX_new(DTLS_client_method());
+            OPENSSL_assert(resume_server_ctx != NULL
```

Comment:
Yep, I'll do that before submit (I won't bother re-pushing here.)

I've actually been meaning to clean all these up with a test-specific shorthand along the lines of

```
ASSERT_NOTNULL(ctx = SSL_CTX_new(DTLS_server_method()));
```